packages/tpetra/core/src/Tpetra_Details_EquilibrationInfo.hpp

```diff
@@ -138,21 +138,27 @@ struct EquilibrationInfo {
   void
   assign (const EquilibrationInfo<ScalarType, SrcDeviceType>& src)
   {
+  std::terminate();
```

Comment:
Don't think we want to keep the std::terminate

Reply:
fixed - BTW I won't plan to use this PR for any submission, just for discussion. I'll update the PR as I merge other PR's so the DEEP_COPY REVIEW comments will only exist for unprocessed deep_copy calls.